packages/jimp/types/test.ts

```diff
@@ -1,4 +1,5 @@
 import * as Jimp from "jimp";
+import { ColorActionName } from "@jimp/plugin-color";
```

Comment:
I'm not sure this would actually work in runtime code. Our TS is .d.ts only so it doesn't effect the actual JS exported by the package so the enum wouldn't exist.

I think we could keep the `ColorActionName` type the same but export an object from actual JS so this works as expected

Reply:
Thanks for spotting!